src/lib.rs

```diff
@@ -290,7 +290,7 @@ impl DocAddress {
 ///
 /// The id used for the segment is actually an ordinal
 /// in the list of `Segment`s held by a `Searcher`.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Do we need serialize for `DocAddress`?

Reply:
At the moment yes, since the aggregators return `DocAddress` and associated scores. Is there another way to go about it?